### ISSUE TITLE
Snap not working when Phantom Wallet is also present

### ISSUE THREAD

**Issue by jhernandezb** [strangelove-ventures/wadesta#152]
Looks like it similar to https://github.com/graz-sh/graz/issues/132 when Phantom Wallet is installed and trying to connect the wallet is showing that is not installed.

**Comment by codingki** [comment]
Yeah phantom wallet also injecting and interrupt metamask window object
